### PR TITLE
First batch for march 2020

### PR DESCRIPTION
**Pull request by organizejs**
#299 #300 #303 #301 #323 #325 #321

### CODE REVIEW

File: actions/js_02.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+  * date: 3/13/2020
+  * sources: https://techcrunch.com/2020/03/13/online-code-collaboration-tool-glitch-votes-to-unionize/, https://cwa-union.org/news/releases/tech-workers-app-developer-glitch-vote-form-union-and-join-cwa-organizing-initiative
+  * actions: union_representation
+  * struggles: na
```
Comment (organizejs) [inline actions/js_02.md]: i added a `na` field here since the kind of struggle was not stated

File: actions/js_09.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+- date: 3/19/2020
+- sources: https://www.bloomberg.com/news/articles/2020-03-19/google-agrees-to-worker-council-in-europe-amid-recurring-discord
+- actions: union_representation
```
Comment (organizejs) [inline actions/js_09.md]: we dont have `formed worker council` as a type of action

Comment (nataliyaned) [reply actions/js_09.md]: legal_action?

File: actions/js_11.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+- date: 3/19/2020
+- sources: https://www.businessinsider.com/ice-mijente-microsoft-coronavirus-crisis-2020-3, https://github.com/msworkers/against.ice
+- actions: open_letter
+- struggles: ethics
+- employment_types: white_collar_workers, in_house_workers
+- description: Mijente, an immigrant rights group is calling on ICE to stop hunting down undocumented immigrants amid the coronavirus crisis. In an open letter published on Github, Microsoft workers backed mijente's campaign, calling on tech companies not to share their technologies with ICE.
+- locations: usa, online
+- companies: microsoft
+- tags: ice, cbp, immigration, coronavirus
```
Comment (organizejs) [inline actions/js_11.md]: cbp is not actually involved here, so i'll remove it

**Pull request merged**